### DISCUSSION TITLE
In V5, how to apply styles to tooltips crosshairs

### DISCUSSION THREAD

**Discussion by clmz** [antvis/G2#6051] [Q&A] [answered]
How can I apply styles (here: change the stroke color) to the tooltips crosshairs?
The doc says:
![image](https://github.com/antvis/G2/assets/47241471/cbbc91c5-ab05-415e-8c35-6d5e5d5a8559)

Can someone please give an example to this?

**Comment by hustcc** [comment]
Here is an [issue](https://github.com/antvis/G2/issues/6063).

**Reply by clmz** [reply, marked as answer]
@hustcc 
Thank you, but - with all due respect - how the hell shall someone figure out how this works without any documentation?

**Reply by pearmini** [reply]
Here is the docs: https://g2.antv.antgroup.com/spec/interaction/tooltip

<img width="842" alt="image" src="https://github.com/antvis/G2/assets/49330279/65d5487c-4721-4c4e-b7df-b908631d77af">
